modules/label/parser.go

```diff
@@ -36,22 +37,25 @@ func (err ErrTemplateLoad) Error() string {
 // GetTemplateFile loads the label template file by given name,
 // then parses and returns a list of name-color pairs and optionally description.
 func GetTemplateFile(name string) ([]*Label, error) {
-	data, err := options.Labels(name + ".yaml")
-	if err == nil && len(data) > 0 {
-		return parseYamlFormat(name+".yaml", data)
-	}
-
-	data, err = options.Labels(name + ".yml")
-	if err == nil && len(data) > 0 {
-		return parseYamlFormat(name+".yml", data)
-	}
-
-	data, err = options.Labels(name)
-	if err != nil {
-		return nil, ErrTemplateLoad{name, fmt.Errorf("GetRepoInitFile: %w", err)}
+	if filepath.Ext(name) == ". yaml" {
```

Comment:
Some questions:

1. `". yaml"`:  a space in it?
2. Gitea has builtin `Advanced.yaml`, if a user puts a customized `Advance.yml`, then still the builtin `Advanced.yaml` is used?

Reply:
1. Typo (will correct )
2.  Good question. I need to check but the built-in Advanced.yaml isn't in the custom options directory. It does raise the question that if some adds foo.yaml and foo.yml ... What should be used